service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ConditionMapper.java

```diff
@@ -61,21 +76,7 @@ public Condition mapCondition(POCDMT000002UK01ClinicalDocument1 clinicalDocument
                     if (component.getSection() != null) {
                         if (component.getSection().isSetLanguageCode()) {
                             if (component.getSection().getLanguageCode().isSetCode()) {
-                                Language.fromCode(component.getSection().getLanguageCode().getCode()).ifPresent(language -> {
-                                    Coding coding = new Coding();
-                                    if (!StringUtils.isBlank(language.getDisplay())) {
-                                        coding.setDisplay(language.getDisplay());
-                                    }
-                                    if (!StringUtils.isBlank(language.getCode())) {
-                                        coding.setCode(language.getCode());
-                                    }
-                                    if (!StringUtils.isBlank(language.getSystem())) {
-                                        coding.setSystem(language.getSystem());
-                                    }
-                                    if (coding.hasCode() || coding.hasSystem() || coding.hasDisplay()) {
-                                        condition.setCode(new CodeableConcept(coding));
-                                    }
-                                });
+                                condition.setCode(new CodeableConcept().setText(component.getSection().getLanguageCode().getCode()));
```

Comment:
What is the reason behind this change? It looks like the entire CodeableConcept was populated and now only text is (which was already reported as an issue in other cases)

Reply:
We previously discussed this to remove language enum unless I have misunderstood?